connections/pyproject.toml

```diff
@@ -9,7 +9,7 @@ python = "^3.12"
 
 # Define ACA-Py as an optional/extra dependency so it can be
 # explicitly installed with the plugin if desired.
-acapy-agent = { version = "~1.2.1", optional = true }
+acapy-agent = { git = "https://github.com/openwallet-foundation/acapy.git", branch = "main", optional = true }
```

Comment:
Did you mean to keep this change?  I assume we would want the version vs. main.

Reply:
The version 1.2.1 doesn't have the v1 connections protocol removed. This would be updated when the next release happens.